client/js/lounge.js

```diff
@@ -1,6 +1,7 @@
 // vendor libraries
 import "jquery-ui/ui/widgets/sortable";
 import $ from "jquery";
+import "fuzzy";
```

Comment:
I did try that, and at the time I couldn't get it to work. Now, it seems like it works.
Maybe I didn't test correctly, or maybe now is not 100% what it was during my first attempt, I don't know, but now it seems OK.